docs/uptime/monitoring/install-nagios-4-on-ubuntu-14-04.md

```diff
@@ -0,0 +1,150 @@
+---
+author:
+  name: Linode Community
+  email: docs@linode.com
+description: 'Install Nagios 4 on Ubuntu 14.04'
+keywords: 'nagios,ubuntu,monitoring,install nagios,nagios 4 ubuntu 14.04'
+alias: ['uptime/monitoring/monitor-services-with-nagios-4-on-ubuntu-14-04/']
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+published: ''
+modified: Monday, March 28th, 2016
+modified_by:
+  name: Linode
+title: 'Install Nagios 4 on Ubuntu 14.04'
+contributor:
+  name: Paulo Telles
+  link: http://github.com/paulotfilho
+external_resources:
+ - '[Nagios Website](http://www.nagios.org/)'
+ - '[Nagios Library](http://library.nagios.com/)'
+---
+
+*This is a Linode Community guide. Write for us and earn $250 per published guide.*
+<hr>
+
+A monitoring tool is a key application in a production server. Nagios is a popular tool that provides monitoring and alerting services for your servers, applications and services. Using Nagios and a wide variety of available plugins, you can keep track of the status of your critical services like HTTP, SSH and SMTP.  In this guide, you'll learn how to install Nagios 4 on your Linode.
+
+## Before You Begin
+
+1. In order to run Nagios on your Linode, follow the usual configuration steps for Ubuntu 14.04 from our [Getting Started guide](/docs/getting-started/). 
+
+2. A running [LAMP stack](docs/websites/lamp/lamp-on-ubuntu-14-04) (Linux, Apache, MySQL and PHP stack) is also required. Follow the "[LAMP on Ubuntu 14.04](docs/websites/lamp/lamp-on-ubuntu-14-04)" guide for instructions on how to set up this stack.
+
+3. This guide will use `sudo` wherever possible. Complete the sections of our [Securing Your Server](/docs/security/securing-your-server) to create a standard user account, harden SSH access and remove unnecessary network services.
+
+## Install Nagios
+
+
+### Steps before installation
+
+1.  As always, make sure that your package repository and packages have been updated:
+
+        sudo apt-get update && sudo apt-get upgrade
+
+2.  To run Nagios, you will need to create both a separate user, `nagios,` and a distinct group, `nagcmd` before inserting the user into the group. You are also going to add the Apache user, `www-data`, to the `nagcmd` group in order to run external commands on Nagios through the web interface:
+
+        sudo useradd nagios
+        sudo groupadd nagcmd
+        sudo usermod -a -G nagcmd nagios
+        sudo usermod -G nagcmd www-data
```

Comment:
Nice catch, it's a typo. Every reference about usermod says you need to use `-a` flag with `-G` otherwise the user is removed from any group not listed in the command. Will fix it.